tests/unit/OptionsListUtilsTest.js

```diff
@@ -295,12 +295,44 @@ describe('OptionsListUtils', () => {
         expect(results.recentReports[0].text).toBe('The Flash');
     });
 
-    it('getNewChatOptions()', () => {
+    it('getSearchOptions() with message timeStampOrder', () => {
```

Comment:
When you will do the above change, here new test will be created for each function including that new function.

Reply:
I will update OptionsListUtilsTest test and removing condition timeStampOrder (true-false). Thank you.